share/man/man4/ix.4

```diff
@@ -74,7 +74,8 @@ For information on enabling VLANs, see
 .Sh HARDWARE
 The
 .Nm
-driver supports the following cards:
+driver supports 10Gb Ethernet PCIe adapters based on the 82598EB controller,
```

Comment:
This change should be undone

Reply:
This change is because the entire hardware section is what appears in the hardware release notes.

https://www.freebsd.org/releases/14.1R/hardware/

Reply:
Sorry the line comment is not exact enough, the part I would remove is specifically the `based on the 82598EB controller`.  While it is true in some absolute sense, each of these cards is marketed independently and the rest of the family is closer affined to the 82599 (aka X520) while it does not add any particular value to the documentation.

Reply:
Should we mention 82599 in DESCRIPTION? E.g. `based on the Intel 82598EB and 82599 Intel(R)...`

Reply:
The HARDWARE section would be more appropriate, I would not recommend any generic mention.

The HARDWARE section could be cleaned up/fleshed out a bit based on `ixgbe_vendor_info_array` in `src/sys/dev/ixgbe/if_ix.c`.. I would say something like this is accurate:

- 82598EB
- 82599 (X520) and X520 Bypass
- X540 and X540 Bypass
- X550
- X552
- X553